src/SDKs/ManagedServices/Management.ManagedServices/Microsoft.Azure.Management.ManagedServices.csproj

```diff
@@ -6,7 +6,7 @@
   <PropertyGroup>
     <PackageId>Microsoft.Azure.Management.ManagedServices</PackageId>
     <Description>Microsoft Azure Managed Services Management Library</Description>
-    <Version>1.0.0</Version>
+    <Version>2.0.0</Version>
```

Comment:
No need to version your tests.
So in your next PR remember you don't need to update test project version

Reply:
Thanks for the feedback @shahabhijeet . Pushed the suggested changes.